apps/prairielearn/src/ee/pages/lti13CourseNavigation/lti13CourseNavigation.html.ts

```diff
@@ -0,0 +1,244 @@
+                              </option>`;
+                            })}
+                          </optgroup>
+                        `;
+                      })}
+                    </select>
+                  </div>
+                  <button class="btn btn-primary btn-lg" id="saveButton" disabled>Save</button>
+                </form> `}
+        </main>
+      </body>
+    </html>
+  `.toString();
+}
+
+export function Lti13CourseNavigationNotReady({
+  courseName,
+  resLocals,
+}: {
+  courseName: string;
+  resLocals: Record<string, any>;
+}): string {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../../../pages/partials/head')%>", {
+          ...resLocals,
+          navPage: 'lti13_course_naviation',
+          pageTitle: 'Course',
+        })}
+      </head>
+      <body>
+        ${renderEjs(__filename, "<%- include('../../../pages/partials/navbar'); %>", {
+          ...resLocals,
+          navPage: 'lti13_course_navigation',
+        })}
+        <main class="container mb-4">
+          <h1 class="h1">Welcome to PrairieLearn</h1>
+          <h2 class="h2">... but your course isn't ready yet!</h2>
+
+          <p>An instructor has not yet configured ${courseName} in PrairieLearn.</p>
+          <p>Please come back later.</p>
+        </main>
+      </body>
+    </html>
+  `.toString();
+}
+
+export function Lti13CourseNavigationDone({
+  resLocals,
+  lti13_instance_id,
+}: {
+  resLocals: Record<string, any>;
+  lti13_instance_id: string;
+}): string {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../../../pages/partials/head')%>", {
+          ...resLocals,
+          navPage: 'lti13_course_naviation',
+          pageTitle: 'Course',
+        })}
+      </head>
+      <body>
+        ${renderEjs(__filename, "<%- include('../../../pages/partials/navbar'); %>", {
+          ...resLocals,
+          navPage: 'lti13_course_navigation',
+        })}
+        <main class="container mb-4">
+          <h1 class="h1">Welcome to PrairieLearn</h1>
+
+          <p>
+            <strong>OK, you're all set.</strong> Next time you (or students) click on the link in
+            your LMS, they will be taken directly to your PrairieLearn course instance.
+          </p>
+          <ul>
+            <li>
+              The course instance <code>allowAccess</code> rules still apply and may need to be
+              configured for the course.
+            </li>
+          </ul>
+
+          <p>
+            To change this connection, go to your course instance "Settings" in the "LTI 1.3" tab.
+          </p>
+
+          <p>
+            <a
+              href="/pl/lti13_instance/${lti13_instance_id}/course_navigation"
+              class="btn btn-success btn-lg"
+              >Click here to continue on to your course instance</a
+            >
```

Comment:
It's bad practice to put "click here" in links, as this doesn't make sense when read by a screen reader.

```suggestion
            <a
              href="/pl/lti13_instance/${lti13_instance_id}/course_navigation"
              class="btn btn-success btn-lg"
              >
                Continue to your course instance
              </a>
```